src/victory-theme/grayscale.js

```diff
@@ -0,0 +1,114 @@
+const grayscaleBlack = "#252525";
+const fontStack = "Poppins, 'Century Gothic', 'Trebuchet MS', sans-serif";
+const letterSpace = 0.2;
+const labelFontSize = 10;
+const dataTickLabelPadding = 5;
+const strokeCap = "round";
+const colors = ["#252525", "#636363", "#969696", "#bdbdbd", "#d9d9d9"];
+const baseLabelStyles = {
+  fontFamily: fontStack,
+  fontSize: labelFontSize,
+  letterSpacing: letterSpace,
+  padding: dataTickLabelPadding,
+  fill: grayscaleBlack
+};
+
+export default {
+  line: {
+    data: {
+      stroke: grayscaleBlack,
+      strokeWidth: 2,
+      fill: "none",
+      opacity: 1
+    },
+    labels: Object.assign(baseLabelStyles,
```

Comment:
`Object.assign` modifies the target object. You want `{}` as your first argument so that you don't modify `baseLabelStyles`